test/end-to-end/multi-supervisor/testcases/rolling_update/run_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This is a simple "happy path" test of a rolling update.
+# We will load services on two nodes to achieve quorum and
+# then promote an update and expect the new release to show
+# up after waiting 15 seconds. Note: we set HAB_UPDATE_STRATEGY_FREQUENCY_MS
+# to 3000 in the docker-compose.override.yml.
+
+set -xeuo pipefail
+
+hab pkg install core/jq-static -b
+
+test_channel=rolling-$(date '+%s%3N')
+test_ident=habitat-testing/nginx/1.17.4/20191115184838
+
+hab pkg promote ${test_ident} "${test_channel}"
+
+hab svc load habitat-testing/nginx --topology leader --strategy rolling --channel "${test_channel}" --remote-sup=alpha.habitat.dev
+hab svc load habitat-testing/nginx --topology leader --strategy rolling --channel "${test_channel}" --remote-sup=beta.habitat.dev
+
+cleanup () {
+    hab bldr channel destroy "${test_channel}" --origin habitat-testing
+}
+
```

Comment:
We should set up a signal trap that ensures that we run this whether the test passes or not, e.g., 

```sh
trap cleanup INT TERM EXIT
```
(You could then remove the calls to `cleanup` when you test to see if the correct services are running, and at the very bottom of the script.)